.github/workflows/test.yml

```diff
@@ -8,10 +8,10 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
+      max-parallel: 4
```

Comment:
Did you mean 4 instead of 2?

Reply:
Just playing with this right now, so I'll mark it as draft :innocent:

Reply:
Dropped this bit now, we might try to find other optimizations.